tests/v2/validation/snapshot/snapshot.go

```diff
@@ -115,49 +120,49 @@ func snapshotRestore(t *testing.T, client *rancher.Client, clusterName string, e
 	require.Equal(t, initialIngressName, ingressResp.ObjectMeta.Name)
 
 	if isRKE1 {
-		initialSnapshotValues := snapshotRKE1(t, client, podTemplate, deployment, clusterName, clusterID, localClusterID, etcdRestore, isRKE1)
-		restoreRKE1(t, client, initialSnapshotValues, etcdRestore, clusterName, clusterID)
+		restoreSnapshotResp, initialSnapshotValues := snapshotRKE1(t, client, podTemplate, deployment, clusterName, clusterID, localClusterID, etcdRestore, isRKE1)
```

Comment:
said this in the shepherd PR too but this name isn't very intuitive to me.

Reply:
Open to ideas on what to change it to. `clusterResp` isn't really good indication for what is happening here.

Reply:
I'll just update it to cluster.